test/jobs/send_daily_triage_email_job_test.rb

```diff
@@ -8,20 +8,64 @@ def setup
 
   test 'send out daily triage email' do
     def @user.issue_assignments_to_deliver; IssueAssignment.all.limit(1); end
+    def @job.before_email_time_of_day?(*); false; end
 
     assert_kind_of Mail::Message, @job.perform(@user)
   end
 
   test 'does not deliver if no subscriptions' do
     def @user.issue_assignments_to_deliver; []; end
+    def @job.before_email_time_of_day?(*); false; end
 
     refute @job.perform(@user)
   end
 
   test 'does not deliver if email should be skipped' do
     def @user.issue_assignments_to_deliver; IssueAssignment.all.limit(1); end
+    def @job.before_email_time_of_day?(*); false; end
```

Comment:
no need for this line because skip_daily_email? returning true will short circuit the conditional